dockerfile-mode.el

```diff
@@ -165,7 +171,8 @@ If prefix arg NO-CACHE is set, don't cache the image."
   (if (stringp image-name)
       (compilation-start
        (format
-        "%sdocker build %s -t %s %s -f %s %s"
+        "%s%s build %s -t %s %s -f %s %s"
+        dockerfile-mode-command
         (if dockerfile-use-sudo "sudo " "")
```

Comment:
Wrongly placed `dockerfile-mode-command`. It comes *after* `sudo`

Reply:
Changed the place i now comes after sudo